src/main/java/org/ow2/proactive/model/CatalogObject.java

```diff
@@ -71,29 +71,44 @@
 
         private final String commitMessage;
 
-        private ImmutableList<CatalogObjectMetadata> metadata;
+        private ImmutableList<CatalogObjectMetadata> objectKeyValues;
 
         private ImmutableList<String> links;
 
-        private String commitDateTime;
+        private String commitTime;
 
         public Builder(CatalogObject catalogObject) {
-            bucketID = catalogObject.bucketID;
+            bucketID = catalogObject.bucket_id;
             name = catalogObject.name;
             kind = catalogObject.kind;
-            contentType = catalogObject.contentType;
-            commitMessage = catalogObject.commitMessage;
-            metadata = Optional.ofNullable(catalogObject.metadata)
+            contentType = catalogObject.content_type;
+            commitMessage = catalogObject.commit_message;
+            objectKeyValues = Optional.ofNullable(catalogObject.object_key_values)
                                .map(metadata -> ImmutableList.copyOf(metadata))
                                .orElse(new ImmutableList.Builder<CatalogObjectMetadata>().build());
             links = Optional.ofNullable(catalogObject.links)
                             .map(links -> ImmutableList.copyOf(links))
                             .orElse(new ImmutableList.Builder<String>().build());
-            commitDateTime = catalogObject.commitDateTime;
+            commitTime = catalogObject.commit_time;
+        }
+
+        public Builder setObjectKeyValues(ImmutableList<CatalogObjectMetadata> object_key_values){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00117)